sim/side.js

```diff
@@ -540,7 +539,7 @@ class Side {
 	 */
 	chooseTeam(data) {
 		const autoFill = !data;
-		if (autoFill) data = `123456`;
+		if (!data) data = `123456`;
```

Comment:
Ditto

Reply:
This one actually makes <code>data</code> correctly typed as <code>string</code> instead of <code>string | undefined</code> but it's only used on 1 line so I guess we can just ignore it for now.

Reply:
The comment is not a bad compromise

Reply:
Alright, I added that instead.